### DISCUSSION TITLE
search_catalog_items returns error when includedData='attributes' is present

### DISCUSSION THREAD

**Discussion by Foxtrot0711** [saleweaver/python-amazon-sp-api#803] [FAQ]
**Describe the bug**
"search_catalog_items" function in 'CatalogItems' Endpoint returns an error when 'attributes' in included in 'includedData'
Error:
[{'code': 'InvalidInput', 'message': "Invalid 'includedData' requested.", 'details': ''}]

but it works fine with "get_catalog_item"

due to 
https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#includeddata-subgroup-2
'attributes' is valid for both search_catalog_items and get_catalog_item

**To Reproduce**

res= client.search_catalog_items (marketplaceIds='A1PA6795UKMFR9', keywords='B01LTHMF1Y', identifiersType='ASIN',includedData='attributes,identifiers,summaries,images,productTypes') 

**Expected behavior**
should not throw an error but return 'attributes' in the result

**Desktop (please complete the following information):**
Windows 10, python 3.10

**Additional context**
It would be very nice if 'attributes' would include 'Creator' or 'Artist'
like 'get_item' in the 'Catalog' Endpoint does in 'AttributeSets'

**Comment by ClaytonSmith** [comment]
Any fixes or work arounds for this?

**Reply by Foxtrot0711** [reply]
I works with Catalog Items API v2022-04-01, not with Catalog Items API v2020-12-01.
but the python library default to v2020-12-01
so you must explicitly set version='2022-04-01'
eg 
client = CatalogItems (credentials=..., marketplace=..., version='2022-04-01')
